src/snappy_device_agents/devices/muxpi/muxpi.py

```diff
@@ -282,6 +292,8 @@ def create_user(self, image_type):
                 cmd = f"sudo cp {remote_tmp}/user-data {base}/system-boot/"
                 self._run_control(cmd)
                 self._configure_sudo()
+            if image_type == "tegra":
+                self._configure_sudo()
```

Comment:
If all it needs to do is run _configure_sudo, then you probably want a `return` here?

Reply:
Thanks, I was hesitating between doing that and sticking to the logic of other image types... But you are right, I still can do that for this case at least.